any_parser/any_parser.py

```diff
@@ -83,10 +101,17 @@ def extract(
         if extract_args is not None and isinstance(extract_args, dict):
             payload["extract_args"] = extract_args
 
+        if model == ModelType.BASE:
+            url = self._sync_url
+        else:
+            url = self._sync_refined_url
+            if model == ModelType.PRO:
+                payload["quick_mode"] = True
```

Comment:
let's be explicit for elif model == ModelType.Pro and elif model == ModelType.Ultra, and else to raise an exception.